device/bgp/bgprouterange.yaml

```diff
@@ -27,6 +27,25 @@ components:
         ext_communities:
           description: >-
             Optional Extended Community settings.
+            The Extended Communities Attribute is a transitive optional BGP attribute, with the Type Code 16. Community and Extended Communities  attributes
+            to trigger routing decisions, such as acceptance, rejection,  preference, or redistribution. An extended community is an 8-Bytes value.
+            It is also divided into two main parts. The first 2 Bytes of the community encode a type and sub-type fields and the last 6 Bytes carry a unique set
+            of data in a format defined by the type and sub-type field. Extended communities provide a larger  range for grouping or categorizing communities.
+            When type is administrator_as_2octet or administrator_as_4octet, the valid sub types are route target and origin. The valid value for 
+            administrator_as_2octet and administrator_as_4octet type is either two byte AS followed by four byte local administrator id or four byte AS followed by two 
+            byte local administrator id. 
+            When type is administrator_ipv4_address the valid sub types are route target and origin. The valid value for 
+            administrator_ipv4_address is an four byte IPv4 address followed by two byte local administrator id. 
+            When type is opaque, valid sub types are color and encapsulation. When sub type is color, first two bytes of the value field contain flags and last four bytes 
+            contains the value of the color. When sub type is encapsulation the first four bytes of value field are reserved and last two bytes carries the tunnel type from 
+            IANA's "ETHER TYPES" registry e.g IPv4 (protocol type = 0x0800), IPv6 (protocol type = 0x86dd), and MPLS (protocol type = 0x8847).
+            When type is administrator_as_2octet_link_bandwidth the valid sub type will be extended_bandwidth. The first two bytes of the value
+            field will carry the AS and the last four bytes the bandwidth in IEEE floating point format. 
```

Comment:
contains the AS number ..... four bytes contain the bandwidth...

Reply:
changed